### ISSUE TITLE
[Bug Report][3.9.5] Accessibility: VTextField input element have role but not required aria attributes (following #21903)

### ISSUE THREAD

**Issue by Dyskal** [vuetifyjs/vuetify#22017]
### Environment
**Vuetify Version:** 3.9.5
**Vue Version:** 3.5.21

### Expected Behavior
The input element inside the `<v-select>` should have role="combobox" and aria-expanded & aria-controls attributes.
https://dequeuniversity.com/rules/axe/4.10/aria-required-attr?application=playwright

### Actual Behavior
The input element have the role but not the required combobox attributes.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpdj0EKwjAQRa8SZpNNTGiLiCUKnsFl00UMIxRiDE3sRry7SWrBdvf+Y/gz070hjEZcvOfTC6EFGfHhrY54Vo4QOe209wVLME8X9eBw/KkiA1o0kVh9Q3tScC1RAWmH1BWS6WimijJSoF6goX0aE0u92PZnM6+X4u8s+DBo+J7XFWQ48gOwu7YB2eqX2fVfoMZGNQ==)

### Other comments
There was no problem before #21903. Those aria attributes must have been forgotten.
